### ISSUE TITLE
[tool_crash] RPCError

### ISSUE THREAD

**Issue by gopikumar443** [flutter/flutter#158978]
## Command
```sh
flutter run
```

## Steps to Reproduce
1. ...
2. ...
3. ...

## Logs
RPCError
```console
#0      VmService._call (package:vm_service/src/vm_service.dart:1774:7)
#1      VmService.callServiceExtension (package:vm_service/src/vm_service.dart:1726:14)
#2      VmService.callMethod (package:vm_service/src/vm_service.dart:1717:12)
#3      FlutterVmService.callMethodWrapper (package:flutter_tools/src/vmservice.dart:501:28)
#4      FlutterVmService.getFlutterViews (package:flutter_tools/src/vmservice.dart:913:51)
#5      HotRunner.attach (package:flutter_tools/src/run_hot.dart:304:63)
<asynchronous suspension>
#6      RunCommand.runCommand (package:flutter_tools/src/commands/run.dart:867:27)
<asynchronous suspension>
#7      FlutterCommand.run.<anonymous closure> (package:flutter_tools/src/runner/flutter_command.dart:1408:27)
<asynchronous suspension>
#8      AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:153:19)
<asynchronous suspension>
#9      CommandRunner.runCommand (package:args/command_runner.dart:212:13)
<asynchronous suspension>
#10     FlutterCommandRunner.runCommand.<anonymous closure> (package:flutter_tools/src/runner/flutter_command_runner.dart:420:9)
<asynchronous suspension>
#11     AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:153:19)
<asynchronous suspension>
#12     FlutterCommandRunner.runCommand (package:flutter_tools/src/runner/flutter_command_runner.dart:364:5)
<asynchronous suspension>
#13     run.<anonymous closure>.<anonymous closure> (package:flutter_tools/runner.dart:130:9)
<asynchronous suspension>
#14     AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:153:19)
<asynchronous suspension>
```
```console
[32m[✓][39m Flutter (Channel stable, 3.24.0, on Ubuntu 20.04.5 LTS 5.15.0-122-generic, locale en_IN)
    [32m•[39m Flutter version 3.24.0 on channel stable at /usr/bin/flutter
    [32m•[39m Upstream repository https://github.com/flutter/flutter.git
    [32m•[39m Framework revision 80c2e84975 (4 months ago), 2024-07-30 23:06:49 +0700
    [32m•[39m Engine revision b8800d88be
    [32m•[39m Dart version 3.5.0
    [32m•[39m DevTools version 2.37.2

[32m[✓][39m Android toolchain - develop for Android devices (Android SDK version 34.0.0)
    [32m•[39m Android SDK at /home/gopi_kumar/Android/Sdk
    [32m•[39m Platform android-34, build-tools 34.0.0
    [32m•[39m ANDROID_HOME = /home/gopi_kumar/Android/Sdk
    [32m•[39m Java binary at: /snap/android-studio/161/jbr/bin/java
    [32m•[39m Java version OpenJDK Runtime Environment (build 17.0.10+0-17.0.10b1087.21-11609105)
    [32m•[39m All Android licenses accepted.

[32m[✓][39m Chrome - develop for the web
    [32m•[39m Chrome at google-chrome

[32m[✓][39m Linux toolchain - develop for Linux desktop
    [32m•[39m clang version 10.0.0-4ubuntu1
    [32m•[39m cmake version 3.16.3
    [32m•[39m ninja version 1.10.0
    [32m•[39m pkg-config version 0.29.1

[32m[✓][39m Android Studio (version 2024.1)
    [32m•[39m Android Studio at /snap/android-studio/161
    [32m•[39m Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    [32m•[39m Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    [32m•[39m Java version OpenJDK Runtime Environment (build 17.0.10+0-17.0.10b1087.21-11609105)

[32m[✓][39m VS Code (version 1.92.0)
    [32m•[39m VS Code at /usr/share/code
    [32m•[39m Flutter extension version 3.100.0

[32m[✓][39m Connected device (3 available)
    [32m•[39m sdk gphone64 x86 64 (mobile) • emulator-5554 • android-x64    • Android 14 (API 34) (emulator)
    [32m•[39m Linux (desktop)              • linux         • linux-x64      • Ubuntu 20.04.5 LTS 5.15.0-122-generic
    [32m•[39m Chrome (web)                 • chrome        • web-javascript • Google Chrome 127.0.6533.99

[33m[!][39m Network resources
    [31m✗[39m A network error occurred while checking "https://pub.dev/": Failed host lookup: 'pub.dev'
    [31m✗[39m A network error occurred while checking "https://storage.googleapis.com/": Failed host lookup: 'storage.googleapis.com'
    [31m✗[39m A network error occurred while checking "https://maven.google.com/": Failed host lookup: 'maven.google.com'
    [31m✗[39m A network error occurred while checking "https://cocoapods.org/": Failed host lookup: 'cocoapods.org'
    [31m✗[39m A network error occurred while checking "https://github.com/": Failed host lookup: 'github.com'

[33m![39m Doctor found issues in 1 category.

```

## Flutter Application Metadata
**Type**: app
**Version**: 1.0.0+1
**Material**: true
**Android X**: false
**Module**: false
**Plugin**: false
**Android package**: null
**iOS bundle identifier**: null
**Creation channel**: stable
**Creation framework version**: 80c2e84975bbd28ecf5f8d4bd4ca5a2490bfc819
### Plugins
fluttertoast-8.2.8

**Comment by danagbemava-nc** [comment]
This has been fixed on stable 3.24.2. Kindly upgrade to the latest version of flutter stable 3.24.5. 

Closing this since it has already been fixed. 

Thank you

**Comment by gopikumar443** [comment]
Thanks for the response...
